billing/details.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Update the contact information
+description: Information on how update contact information in Docker Hub
+keywords: payments, billing, subscription, contact
+---
+
+Find information on how to update the contact information for your personal account or for an organization. 
```

Comment:
**chore:** Can you please highlight that this information appears on all billing invoices?